LCM/scripts/OMS_MetaConfigHelper.py

```diff
@@ -206,9 +207,13 @@ def source_file(filename):
     commandToRun = "/opt/microsoft/omsconfig/Scripts/SetDscLocalConfigurationManager.py -configurationmof " + metamof_path
 
 # Apply the metaconfig using SetDscLocalConfiguration
-retval = os.system(commandToRun)
-if (retval != 0):
-    exitWithError("Error on running command: " + commandToRun)
+proc = subprocess.Popen(commandToRun, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True, close_fds=True)
+exit_code = proc.wait()
```

Comment:
>wait [](start = 17, length = 4)

proc.communicate() is preferred method to receive stdout and stderr data from subprocess pipe.  check the documentation  [https://docs.python.org/dev/library/subprocess.html#subprocess.Popen.wait](https://docs.python.org/dev/library/subprocess.html#subprocess.Popen.wait)

Reply:
fixed